README.md

```diff
@@ -4,6 +4,9 @@
 
 > Client-side GraphQL convenience wrapper for the WordPress REST API
 
+## What
+This project is a wrapper for the [WordPres REST API](http://v2.wp-api.org/) that let's you query your site content using [GraphQL](http://graphql.org/) and [TypeScript](https://www.typescriptlang.org/). We like the simplicity and flexibility of GraphQL queries rather than the endless chains of REST queries the traditional API requires. As a bonus, the TypeScript implementation made it easy to integrate into our [React](https://facebook.github.io/react/) frontend. Though the library can be used on the backend as well.
```

Comment:
Thanks for the feedback. A couple comments:
1. `let's` -> `lets`
2. http://v2.wp-api.org/ is not the official URL. It moved to https://developer.wordpress.org/rest-api/.
3. I think that the intro caters to a demographic that should not be using the library: Inexperienced JS users. People who are reading it should be fully aware of what GraphQL and React are, so I think it's just over-winded to mention it.
4. I'd remove TypeScript from the intro because that part doesn't make any difference to the user (in the "advanced users" section, it brings up how to import and use the type defs).